docs/install/installEverest.md

```diff
@@ -46,10 +37,25 @@ To install and provision Percona Everest to Kubernetes:
         everestctl install --namespaces dev,prod --operator.mongodb=true --operator.postgresql=true --operator.xtradb-cluster=true --skip-wizard
         ```
 
-    !!! warning "Important"
-        Ensure to copy the authorization token displayed on the terminal in this step. You will need this token to log in to the Percona Everest UI.    
+    To gain a deeper understanding of how to install different operators in various namespaces, refer to the [configure multiple namespaces](../use/multi-namespaces.md#configure-multiple-namespaces) section.
+
+
+3. Update the password for the `admin` user:
+
+    ```sh
+    everestctl accounts set-password --username admin
+    ```
+
+    !!! info "Important"
+
+        - You can retrieve the automatically generated password by running the `everestctl accounts initial-admin-password` command. However, this password isn't stored securely.
+
+        - Do not modify the root user's password in the database, as this could potentially break your cluster.
```

Comment:
This point shouldn't be part of the installation page. This refers to the DB users so it's better to have it in the [DB Provisioning page](https://everest-release-branch-pr-132.onrender.com/use/db_provision.html)

Reply:
Done.

Reply:
Done